project/scenes/scene_decoder.gd

```diff
@@ -261,11 +261,8 @@ func create_object(object: Dictionary, geometries: Dictionary, materials: Dictio
 	elif "centre" in object && "radius" in object && (!object.has("with_script") || !object.with_script):
 		var centre := get_randomized_vector3(object.centre, rng)
 		var radius: float = get_randomized(object.radius, rng)
-		node.transform.origin = centre + radius * Vector3(
-			cos(rng.randf_range(0, 2 * PI)),
-			sin(rng.randf_range(0, 2 * PI)),
-			0
-		)
+		var phi := rng.randf_range(0, 2 * PI)
+		node.transform.origin = centre + radius * Vector3(cos(phi), sin(phi), 0)
```

Comment:
This looks basically like a scaled and translated unit circle -- a *full* unit circle.

Is there an area around the centre cut out or does this really mean that the circle intersects the planet surrounded by the belt, meaning some objects *might* be created within it?

Reply:
Okay, now I understood.  It is a cylinder with a cut-out centre.